src/Adapter/Presenter/Cart/CartPresenter.php

```diff
@@ -693,6 +693,7 @@ protected function getSettings(): ProductPresentationSettings
             $this->settings->stock_management_enabled = Configuration::get('PS_STOCK_MANAGEMENT');
             $this->settings->showPrices = Configuration::showPrices();
             $this->settings->showLabelOOSListingPages = (bool) Configuration::get('PS_SHOW_LABEL_OOS_LISTING_PAGES');
+            $this->settings->lastRemainingItems = Configuration::get('PS_LAST_QTIES');
```

Comment:
Could you push Configuration to the constructor, please ?

Reply:
Isn't it out of scope of this PR?

Reply:
As it adds a new line, it should be added following coding standards, so with dependy injection.

Reply:
It's not possible as the CartPresenter is used in standalone.
We can't add dependency injection in this one.

Reply:
@PierreRambaud But ModulePresenter or ProductPresenter are not working as standalone

Reply:
Agreed with @PierreRambaud and @atomiix

1. It would make no sense to use a `$configuration` instance for this line while the 7 above rely on the singleton. I prefer an homogenous function that rely on legacy way rather than a function that uses 90% legacy way and 10% modern way.

2. `CartPresenter` is built inside `CartController` using `$presenter = new CartPresenter();` . It's not a service retrieved from a service container. I think this is what @PierreRambaud says when referring to "it's used as standalone". So if you start injecting dependencies in the CartController you must modify how it's built. ModulePresenter is different as it's used in the Back-office, not Front-office, where services are mainly used.

Relying on dependency injection for CartPresenter could be interesting, but in another PR, where you would have to solve the question "how do I inject my dependencies": with service container? with a factory, like ProductPresenterFactory? these 2 questions are far from the scope of this PR